lhotse/dataset/collation.py

```diff
@@ -163,7 +163,18 @@ def collate_audio(
     audio = torch.empty(len(cuts), first_cut.num_samples)
     if executor is None:
         for idx, cut in enumerate(cuts):
-            audio[idx] = _read_audio(cut)
+            try:
+                audio[idx] = _read_audio(cut)
+            except AudioLoadingError as e:
+                warnings.warn(
+                    f"AudioLoadingError for {cut} \nError messages: {e} \nSkipping this cut."
+                )
+                continue
+            except DurationMismatchError as e:
+                warnings.warn(
+                    f"DurationMismatchError for {cut} \nError messages: {e} \nSkipping this cut."
+                )
+                continue
     else:
         for idx, example_audio in enumerate(executor.map(_read_audio, cuts)):
```

Comment:
@pzelasko Any suggestions to handle exceptions in this multithread executor.

Reply:
hmm, one option is to do the following: ` _read_audio` could return `Union[torch.Tensor, Exception]` and the code in this loop could check `if isinstance(audio_or_error, Exception): raise audio_or_error`